tests/gdx-tests-gwt/webapp/index.html

```diff
@@ -1,25 +1,25 @@
-<!doctype html>
-<html>
-  <head>
-    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
-    <title>libGDX GWT Test Runner</title>
-  </head>
-  <body>
-    <h1>libGDX GWT Test Runner</h1>
-    UI should load here shortly:<br>
-    <div id="embed-com.badlogic.gdx.tests.gwt.GdxTestsGwt"></div>
-    <small>This page is from <code>tests/gdx-tests-gwt/war/index.html</code>. Enter superdev here:
-    <a class="superdev" href="javascript:%7B%20window.__gwt_bookmarklet_params%20%3D%20%7B'server_url'%3A'http%3A%2F%2Flocalhost%3A9876%2F'%7D%3B%20var%20s%20%3D%20document.createElement('script')%3B%20s.src%20%3D%20'http%3A%2F%2Flocalhost%3A9876%2Fdev_mode_on.js'%3B%20void(document.getElementsByTagName('head')%5B0%5D.appendChild(s))%3B%7D">&#8635;</a>
-    </small>
-    <script type="text/javascript" language="javascript" src="com.badlogic.gdx.tests.gwt.GdxTestsGwt/com.badlogic.gdx.tests.gwt.GdxTestsGwt.nocache.js">
-window.onkeydown =
-  function(event) {
-    // prevent all navigation keys except the space key
-    if([9, 33, 34, 35, 36, 37, 38, 39, 40].indexOf(event.keyCode) > -1) {
-        event.preventDefault();
-        return false;
-    }
-  };
-    </script>
-  </body>
-</html>
+<!doctype html>
+<html>
+  <head>
+    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
+    <title>libGDX GWT Test Runner</title>
+  </head>
+  <body>
+    <h1>libGDX GWT Test Runner</h1>
+    UI should load here shortly:<br>
+    <div id="embed-com.badlogic.gdx.tests.gwt.GdxTestsGwt"></div>
+    <small>This page is from <code>tests/gdx-tests-gwt/war/index.html</code>. Enter superdev here:
```

Comment:
Is this path incorrect now? I know the webapp folder gets copied to war, but if people are looking for this file to edit it, they should go to webapp.